banhmi/prisma/seed.ts

```diff
@@ -36,6 +36,105 @@ async function createUsers() {
       hashedPassword: sha256(sha256('password' + salt) + process.env.PEPPER),
     },
   });
+  salt = getRandomString(50);
```

Comment:
This one is ok for now but i will create a function to create user in another PR